ovn-fake-multinode-utils/playbooks/run-tester.yml

```diff
@@ -0,0 +1,20 @@
+- name: Run the tester
+  hosts: ovn_hosts
+  become: true
+  tasks:
+  - name: Start tester container
+    when: ovn_tester is defined
+    shell: |
+      docker run -dt --name=ovn-tester --hostname=ovn-tester --privileged ovn/ovn-tester
+
+  - name: Copy files to the tester container
+    when: ovn_tester is defined
+    shell: |
+      docker cp {{ test_file }} ovn-tester:/test-scenario.yml
```

Comment:
This only works if the tester is the same machine as the orchestrator.  Otherwise we need to move the test file to the tester too.

Reply:
Ah, good catch.